.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,43 @@
+name: Feature or enhancement
+description: Submit a proposal for a new CPython feature or enhancement
+labels: ["type-feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Proposing a feature to CPython?
+
+        You'll need to demonstrate widespread support for your idea among the community.
+
+        Major feature proposals should generally be discussed on [Discourse](https://discuss.python.org/c/ideas/6) before opening a GitHub issue.
+        Wait until it's clear that most people support your idea before filling in this form.
+  - type: checkboxes
+    attributes:
+      label: Has this already been discussed elsewhere?        
+      options:
+        - label: I have already discussed this feature proposal on Discourse
+        - label: This is a minor feature, which does not need previous discussion elsewhere
+  - type: textarea
+    attributes:
+      label: Links to previous discussion of this feature
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Summary of proposal
+      description: A clear and concise summary of your proposal
+    validations:
+      required: true
+  - type: textarea
```

Comment:
I wonder if here we could also get rid of a one or two texteares to make the form simpler.

Reply:
I converted the first `textarea` into an `input`